src/main/java/codeu/model/data/User.java

```diff
@@ -58,4 +65,32 @@ public String getPasswordHash() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Returns the number of messages sent by this User. */
+  public long getNumMessages() {
+    return numMessages;
+  }
+
+  /** Returns the number of messages sent by this User. */
+  public long getIncNumMessages() {
+    this.numMessages = numMessages + 1; 
+    return numMessages;
+  }
+
+  /** Returns the number of words typed by this User. */
+  public long getNumWords() {
+    return numWords;
+  }
+
+  /** Returns the number of words sent by this User. */
+  public long getIncNumWords(long words) {
+    this.numWords = numWords + words; 
```

Comment:
Let's change this to `incrementNumWords()` for the same reasons defined in the above comment?